Dockerfile

```
@@ -16,6 +16,10 @@ COPY ./rust-connector-sdk .
 RUN cargo build --release
 
 FROM debian:buster-slim as connector
+RUN apt-get update \
+ && DEBIAN_FRONTEND=noninteractive \
+    apt-get install --no-install-recommends --assume-yes\
+    libssl-dev
```

Adding `set -ex` makes it easier to figure out which line broke when something goes wrong.

```suggestion
RUN set -ex; \
    apt-get update; \
    DEBIAN_FRONTEND=noninteractive \
      apt-get install --no-install-recommends --assume-yes \
      libssl-dev
```